tools/monitoring/main.go

```diff
@@ -57,3 +76,40 @@ func hello(w http.ResponseWriter, r *http.Request) {
 	errorPatterns := yamlFile.CollectErrorPatterns()
 	fmt.Fprintf(w, "error patterns collected from yaml:%s", errorPatterns)
 }
+
+func testCloudSQLConn(w http.ResponseWriter, r *http.Request) {
```

Comment:
move sql functions into a separate file.

Reply:
Moved the actual test connection logic to mysql. This function is now just a wrapper around the TestConn and is used to exposed 'test-conn' path to test connection once the app is started.